style_rwd.css

```diff
@@ -1,22 +1,67 @@
 @media (max-width: 767px){
     img.main_photo {
         display: none;
+        width:auto;
     }
-    .catch_phrase{
-        max-width: 767px;
+    .section_hero {
+        padding-bottom: 42px;
+    }
+    body{
+        width: auto;
     }
     .column{
+        width: auto;
         align-items: center;
         justify-content: center;
     }
-    .invite_phrase{
+    .catch_phrase .invite_phrase{
         text-align: center;
-        font-size: 5px;
+        font-size: 40px;
         font-weight: 30;
-        max-width: 767px;
+        margin-bottom: 0;
+        width: auto;
+    }
+    .button_container {
+        justify-content: center;
+        width: auto;
+        margin: 0;
     }
-    .btn{
+    .btn {
+        border-radius: 32px;
+        padding: 5px 16px;
         font-size: 12px;
     }
-}
+    .section_container {
+        min-height: 400px;
+        justify-content: center;
+    }
+    .catch_phrase {
+        flex-direction: column;
+        width: auto;
+        line-height: 95px;
+    }
+
+    .flex_column {
+        flex-direction: column;
+    }
+
+    .section_heading {
+        margin: auto;
+        font-size: 18px;
+        font-weight: 200;
```

Comment:
What font weight 200 stand for?

Reply:
well it was the half of the starting point font weight as it defines how bold text should be... thought it is necessary

Reply:
we keep the same font weight across all devices

Reply:
using new weight = importing new weight, you want to avoid that 😉

Reply:
understood